addons/stock/models/stock_inventory.py

```diff
@@ -348,6 +352,29 @@ def _run_cyclic_inventory_tasks(self, company_id=False):
                                       'location_ids': location})
             self.create(location_vals)
 
+    @api.model
+    def _run_conflict_inventory_tasks(self):
```

Comment:
We could put it in scheduler to avoid creating too much object at once if the user only opens inventory tab once a year. But it will probably cost useless computation every day. What do you think about it?

Reply:
I think you're right that we would save some performance at time of tab opening, but in addition to the useless computation every day we're also potentially:
1. unlinking inventories more often than needed [not sure if this matters] and 
2. making the "Inventory Date" earlier [probably doesn't matter unless the user gets confused, although could be useful for knowing the earliest date the problem has consistently existed].

Either way from a UX point of view it's probably better to always be faster for the user so I'll move it.